TFproviderTest.yml

```diff
@@ -171,29 +171,30 @@ pipelines:
             - cat GNUmakefile | sed -e "s/darwin_amd64/linux_amd64/g" > GNUmakefile.tmp
             - cp GNUmakefile.tmp GNUmakefile && rm GNUmakefile.tmp
             - cat GNUmakefile
-            - cat sample.tf | sed -e "s/version =.*/version = \"${NEXT_PROVIDER_VERSION}\"/g" > sample.tf.tmp
-            - cp sample.tf.tmp sample.tf && rm sample.tf.tmp
-            - cat sample.tf
+            ## commented out before the first version released. Version is hardcoded to 0.0.1
+            #- cat sample.tf | sed -e "s/version =.*/version = \"${NEXT_PROVIDER_VERSION}\"/g" > sample.tf.tmp
+            #- cp sample.tf.tmp sample.tf && rm sample.tf.tmp
+            #- cat sample.tf
             - echo "Add variables needed to run Terraform Provider Xray"
-            - export ARTIFACTORY_URL=${jdp_url} && echo ${ARTIFACTORY_URL}
+            - export JFROG_URL=${jdp_url} && echo ${JFROG_URL}
             - export ARTIFACTORY_USERNAME=admin
             - export ARTIFACTORY_PASSWORD=${NEW_RT_PASSWORD} && echo ${ARTIFACTORY_PASSWORD}
             - echo "Get cookie to generate Access token"
             - >-
-              export COOKIES=$(curl -c - "${ARTIFACTORY_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
+              export COOKIES=$(curl -c - "${JFROG_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
                             --header "accept: application/json, text/plain, */*" \
                             --header "content-type: application/json;charset=UTF-8" \
                             --header "x-requested-with: XMLHttpRequest" \
                             -d '{"user":"admin","password":"'"${ARTIFACTORY_PASSWORD}"'","type":"login"}' | grep FALSE)
             - export REFRESHTOKEN=$(echo $COOKIES | grep REFRESHTOKEN | awk '{print $7 }')
             - export ACCESSTOKEN=$(echo $COOKIES | grep ACCESSTOKEN | awk '{print $14 }') # awk returns null on Mac, and the actual key on Ubuntu
             - >-
-              export ACCESS_KEY=$(curl -g --request GET "${ARTIFACTORY_URL}/ui/api/v1/system/security/token?services[]=all" \
+              export ACCESS_KEY=$(curl -g --request GET "${JFROG_URL}/ui/api/v1/system/security/token?services[]=all" \
                                   --header "accept: application/json, text/plain, */*" \
                                   --header "x-requested-with: XMLHttpRequest" \
                                   --header "cookie: ACCESSTOKEN=${ACCESSTOKEN}; REFRESHTOKEN=${REFRESHTOKEN}")
-            - add_run_variables JFROG_ACCESS_TOKEN=${ACCESS_KEY}
-            - echo "Unset ARTIFACTORY_USERNAME and ARTIFACTORY_PASSWORD, acceptance test will use JFROG_ACCESS_TOKEN instead"
+            - export XRAY_ACCESS_TOKEN=${ACCESS_KEY}
```

Comment:
I'd rather continue using `JFROG_ACCESS_TOKEN` to be consistent. But if pipeline doesn't like it, not much we can do.